frontend/package.json

```diff
@@ -50,14 +50,14 @@
     "html-react-parser": "^1.4.0",
     "humanize-duration": "^3.22.0",
     "lodash": "^4.17.21",
-    "next": "^11.1.3",
+    "next": "^12.0.9",
```

Comment:
So looks like we needed v12 to handle redirects - any other breaking changes we should be aware of with the major upgrade? Didn't exhaustively look, but here's the upgrade guide: https://nextjs.org/docs/upgrading

Reply:
No breaking changes :) 
Actually we would have needed that for middleware redirects which we didn't opt for in the end but since there were no breaking changes anywhere I just left it in

Reply:
No changes that break out code to be precise. There are changes that could be breaking but we don't use any of the affected features :)